layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend

```diff
@@ -2,21 +2,18 @@ inherit kernel-resin
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
+LINUX_VERSION = "4.19.44"
+
 SRC_URI_append = " \
-	file://0001-Revert-cgroup-Disable-cgroup-memory-by-default.patch \
```

Comment:
I see we removed this patch from SRC_URI and haven't removed it as a file? Also, shouldn't this still be needed? I don't expect they reverted it upstream.

Reply:
memory cgroups are activated by default in the new kernel version:
root@f026b3a:~# cat /proc/cgroups 
#subsys_name    hierarchy       num_cgroups     enabled
cpuset  8       4       1
cpu     6       4       1
cpuacct 6       4       1
blkio   5       4       1
memory  9       98      1
devices 2       10      1
freezer 4       4       1
net_cls 7       4       1
pids    3       40      1